tests/models/test_dag.py

```diff
@@ -2029,14 +2029,14 @@ def test_count_number_queries(self, tasks_count):
 
 
 class TestDagDecorator(unittest.TestCase):
+    DEFAULT_DATE = timezone.datetime(2016, 1, 1)
     DEFAULT_ARGS = {
         "owner": "test",
         "depends_on_past": True,
-        "start_date": timezone.utcnow(),
+        "start_date": DEFAULT_DATE,
```

Comment:
Unrelated?

Reply:
It affects `test_dag_param_dagrun_parameterized` and `test_dag_param_resolves` where `execution_date` is `self.DEFAULT_DATE` (2016-01-01), but `execution_date` can not be less than DAG's start_date.